.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+- React-Native tools version: 
+- VSCode version: 
+- OS version: 
+- React Native version: 
+- Output of the Debug Console (View -> Toogle Debug Console): 
```

Comment:
What do you think of adding (If relevant) instead?
I don't want someone to think: It's optional, so I don't need to send it...